tests/datalab/datalab/test_datalab.py

```diff
@@ -171,6 +204,7 @@ def test_get_issues(self, lab, monkeypatch):
             },
         )
         monkeypatch.setattr(lab, "issues", mock_issues)
+        assert lab.issues is lab.data_issues.issues
```

Comment:
I'd either get rid of this line, or add explanatory print statement whenever this assert fails. Otherwise it's unclear what this is testing / what developers should do when this assert fails.